src/modules/poolModule.ts

```diff
@@ -527,7 +528,7 @@ export class PoolModule implements IModule {
    */
   async fetchTicks(params: FetchParams): Promise<TickData[]> {
     let ticks: TickData[] = []
-    let start: number[] = []
+    let start = 0
```

Comment:
left a comment about this change above, this should definitely be verified before merging